en/docs/monitoring-and-insights/logs/view-pdp-logs.md

```diff
@@ -0,0 +1,73 @@
+# View Private Data Plane (PDP) Logs 
+
+Choreo offers the capability to access runtime logs through its console. However, in cases where viewing logs for your PDP is not supported by Choreo, you can still view the runtime logs of your components via the log analyzing engines associated with your PDP.
+
+## Prerequisites
+
+Before you try out this guide, complete the following:
+
+1. Go to [https://console.choreo.dev/](https://console.choreo.dev/), and sign in using your preferred method.
+2. Select your component from **Components Listing**. This will open the overview page of your component.
+3. In the left navigation menu, click **Runtime** under **DevOps**.
+4. Copy the `Release ID` and the `Namespace`. Save it for later.
+
+## View Private Data Plane (PDP) logs with Azure Log Analytics
+
+You can view your PDP logs with Azure Log Analytics by following the steps below: 
+
+1. Go to https://portal.azure.com/.
+2. Follow the [Azure Log Analytics Tutorial](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-tutorial#open-log-analytics) and open log analytics of your relative log analytics workspace.
+3. Copy and paste the query below into the query editor. 
+4. Replace the `startDateTime` and `endDateTime` as required. Replace the '<RELEASE_ID>' and '<NAMESPACE>' with the values you copied by following the steps in the [prerequisites](#prerequisites) section.
+5. Run the query to extract the relevant logs.
+
+```SQL
+let startDateTime = datetime('<END_TIME_STAME EX: 2023-04-10T07:07:31.684Z>');
+let endDateTime = datetime('<END_TIME_STAME EX: 2023-04-21T07:27:31.684Z>');
```

Comment:
```suggestion
let startDateTime = datetime('<START_TIME_STAMP EX: 2023-04-10T07:07:31.684Z>');
let endDateTime = datetime('<END_TIME_STAMP EX: 2023-04-21T07:27:31.684Z>');
```

Reply:
There was a typo in the source document. I fixed it.

Reply:
done